lib/modules/dosomething/dosomething_kudos/dosomething_kudos.module

```diff
@@ -245,7 +245,10 @@ function dosomething_kudos_get_kudos_for_user_by_file($fid) {
 function dosomething_kudos_term_is_selected($reportback_item, $term) {
   // We need to check both array & objects b/c preprocess_node works w/ RB Item objects
   // and preprocess_user works w/ content arrays.
-  if ((is_array($reportback_item) && empty($reportback_item['existing_kids'])) && ! $reportback_item->existing_kids) {
+  if (is_array($reportback_item) && empty($reportback_item['existing_kids'])) {
+    return false;
+  }
+  else if (!is_array($reportback_item) && !$reportback_item->existing_kids) {
```

Comment:
This function only returns a boolean, so there wouldn't be a need. It'd just be cast inside the function and then forgotten cause all it sends back is `true/false`.